homeassistant/components/ezviz/image.py

```diff
@@ -0,0 +1,88 @@
+"""Support EZVIZ last motion image."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.image import Image, ImageEntity, ImageEntityDescription
+from homeassistant.config_entries import (
+    ConfigEntry,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import (
+    AddEntitiesCallback,
+)
+from homeassistant.util import dt as dt_util
+
+from .const import (
+    DATA_COORDINATOR,
+    DOMAIN,
+)
+from .coordinator import EzvizDataUpdateCoordinator
+from .entity import EzvizEntity
+
+_LOGGER = logging.getLogger(__name__)
+GET_IMAGE_TIMEOUT = 10
+
+IMAGE_TYPE = ImageEntityDescription(
+    key="last_motion_image",
+    translation_key="last_motion_image",
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up EZVIZ image entities based on a config entry."""
+
+    coordinator: EzvizDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id][
+        DATA_COORDINATOR
+    ]
+
+    async_add_entities(
+        EzvizLastMotion(hass, coordinator, camera) for camera in coordinator.data
+    )
+
+
+class EzvizLastMotion(EzvizEntity, ImageEntity):
+    """Return Last Motion Image from Ezviz Camera."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, hass: HomeAssistant, coordinator: EzvizDataUpdateCoordinator, serial: str
+    ) -> None:
+        """Initialize a image entity."""
+        super().__init__(coordinator, serial)
+        ImageEntity.__init__(self, hass)
```

Comment:
This is following the pattern from other ezviz entities, but if the super classes don't support cooperative multiple inheritance I think this pattern may be clearer:
```py
        EzvizEntity.__init__(self, coordinator, serial)
        ImageEntity.__init__(self, hass)
```